mscore/instrdialog.cpp

```diff
@@ -519,6 +519,8 @@ void MuseScore::editInstrList()
             else {
                   for (Staff* s : sl) {
                         const LinkedElements* sll = s->links();
+                        if (!sll)
+                              break;
```

Comment:
Why not `continue`? Then all staves which have links would be handled, and the current fix will stop at the first staff which doesn't have links.

Reply:
good question... let's to that

Reply:
done